src/myopic.jl

```diff
@@ -186,3 +188,58 @@ function _entropy(model::BinomialGridModel, obs::BinomialObservation, ::MyopicFa
     
     return argmin(entropies)
 end
+
+function _tauentropy(model::BinomialGridModel, obs::BinomialObservation, ::Myopic)
```

Comment:
Could you add some unit tests for _tauentropy, similar to [here](https://github.com/Theoretical-Neuroscience-Group/BinomialSynapses.jl/blob/OED_HL_test_CG/test/myopic.jl#L42-L138)

Reply:
Done, if I understood correctly, results of the test should be the same for _tauentropy and _entropy, is that correct ?